### DISCUSSION TITLE
Is there a way to use the UPS dashboard without having an internet connected UPS

### DISCUSSION THREAD

**Discussion by maxall41** [carlosedp/cluster-monitoring#150] [General]
Hi,
I'm trying to get the UPS dashboard working but I don't have a UPS with ethernet I have managed to get apcupsd running on my master node so is there a way to connect to that instead of directly connecting to the UPS?

**Comment by badnetmask** [comment]
I will have to do that myself, eventually, but in theory you should be able to figure something out using a "daemonset". That's what Telegraf uses to gather low-level hardware metrics.

Not sure when I will try this myself, but I'll keep monitoring this discussion, and report back if I get to do it.

**Comment by carlosedp** [comment]
I use a raspberry pi to collect the UPS data using apcupsd and publish it so Prometheus can use.
